pkg/kyma/cmd/install.go

```diff
@@ -125,34 +134,42 @@ func (o *InstallOptions) Run() error {
 	return nil
 }
 
-func checkReqs(o *InstallOptions) error {
-	err := internal.CheckKubectlVersion()
-	if err != nil {
-		return err
-	}
-	if o.LocalSrcPath != "" && !o.Local {
-		return fmt.Errorf("You specified 'src-path=%s' without specifying --local", o.LocalSrcPath)
-	}
-	if o.LocalSrcPath == "" {
-		goPath := os.Getenv("GOPATH")
-		if goPath != "" {
-			o.LocalSrcPath = filepath.Join(goPath, "src", "github.com", "kyma-project", "kyma")
-		}
-	}
+func validateFlags(o *InstallOptions) error {
 	if o.Local {
 		if o.LocalSrcPath == "" {
-			return fmt.Errorf("No local 'src-path' configured and no applicable default found, verify if you have exported a GOPATH?")
+			goPath := os.Getenv("GOPATH")
+			if goPath == "" {
+				return fmt.Errorf("No local 'src-path' configured and no applicable default found, verify if you have exported a GOPATH?")
+			}
+			o.LocalSrcPath = filepath.Join(goPath, "src", "github.com", "kyma-project", "kyma")
 		}
 		if _, err := os.Stat(o.LocalSrcPath); err != nil {
 			return fmt.Errorf("Configured 'src-path=%s' does not exist, please check if you configured a valid path", o.LocalSrcPath)
 		}
 		if _, err := os.Stat(filepath.Join(o.LocalSrcPath, "installation", "resources")); err != nil {
 			return fmt.Errorf("Configured 'src-path=%s' seems to not point to a Kyma repository, please verify if your repository contains a folder 'installation/resources'", o.LocalSrcPath)
 		}
+		if o.LocalInstallerDir == "" && strings.HasPrefix(o.LocalInstallerVersion, "PR-") {
+			o.LocalInstallerDir = " "
```

Comment:
Sounds like hidden magic, why only for a PR and why a space? Whatever the reason is, I guess no one else then you will understand it when reading the code...

Reply:
It was supposed to be `eu.gcr.io/.../pr` not space :D. It's a shortcut so you don't have to provide `--installer-dir` when running Kyma from PR.

Reply:
What means "supposed", will you change it?

Reply:
Yes

Reply:
Is it better now?